tests/tools/test_msc.lua

```diff
@@ -87,7 +87,8 @@
 	function suite.ldflags_onLinkTimeOptimizations()
 		flags "LinkTimeOptimization"
 		prepare()
-		test.contains("/GL", msc.getldflags(cfg))
+		test.contains("/GL", msc.getcflags(cfg))
```

Comment:
This test suite is for `ldflags`. `cflags` tests should go into their own suite.

Reply:
I split up the tests